app/Services/Backup/Databases/Neo4jDatabase.php

```
@@ -0,0 +1,177 @@
+            function ($tsx) use ($cypher): void {
+                $tsx->run('CALL apoc.cypher.runMany($cypher, {})', ['cypher' => $cypher]);
+            }
+        );
+
+        return new DatabaseOperationResult(
+            command: null,
+            log: new DatabaseOperationLog('Neo4j APOC restore completed', 'info'),
+        );
+    }
+
+    public function prepareForRestore(string $schemaName, BackupLogger $logger, bool $forceDatabase = false): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schemaName'. (undefined)

(UnusedFormalParameter)